spec/moip2/accounts_api_spec.rb

```diff
@@ -140,4 +140,54 @@
       end
     end
   end
+
+  describe "#exists" do
+    describe "with a registered tax document" do
+      let(:existence_check) do
+        VCR.use_cassette("account_exists") do
+          accounts_api.exists?("436.130.670-21")
+        end
+      end
+
+      it { expect(existence_check).to be true }
+    end
+
+    describe "with a non registered tax document" do
+      let(:existence_check) do
+        VCR.use_cassette("account_doesnt_exist") do
+          accounts_api.exists?("555.000.123-40")
+        end
+      end
+
+      it { expect(existence_check).to be false }
+    end
+  end
+
+  describe "#show" do
+    context "given an existent account" do
```

Comment:
Utilize os context para "contar uma história", comece os contextos com `when` ou `with` e evite deixar a descrição do que o `it` faz, você deve ser capaz de conseguir ler o que ele faz ;)

-----

## Use contexts
Contexts are a powerful method to make your tests clear and well organized. In the long term this practice will keep tests easy to read.

BAD
```
it 'has 200 status code if logged in' do
  expect(response).to respond_with 200
end
it 'has 401 status code if not logged in' do
  expect(response).to respond_with 401
end
```

GOOD
```
context 'when logged in' do
  it { is_expected.to respond_with 200 }
end
context 'when logged out' do
  it { is_expected.to respond_with 401 }
end
```

When describing a context, start its description with "when" or "with".

http://www.betterspecs.org/